src/lib.rs

```diff
@@ -244,6 +252,7 @@ construct_runtime!(
 		CouncilElection: election::{Module, Call, Storage, Event<T>, Config<T>},
 		Council: council::{Module, Call, Storage, Event<T>, Config<T>},
 		Memo: memo::{Module, Call, Storage, Event<T>},
+		DataObjectType: types::{Module, Call, Storage, Event<T>, Config<T>},
```

Comment:
I sort of agree. I mean, I agree with the aesthetics, I just don't know what the best thing to do here is. I think it should be called `DataObjectTypeRegistry`, and the module should be called `registry` to keep things simpler, because that's what the whitepaper said.

WDYT?

Reply:
Pushed a new commit that should address this.